### DISCUSSION TITLE
Youtube Plugin

### DISCUSSION THREAD

**Discussion by CQer789** [e107inc/e107#4656] [Q&A Plugins]
Hello,
I was wondering if someone could explain the Youtube plugin a bit further.
For the time being I have filled in:
In the top field (which says LAN_YOUTUBE_ADMIN_05): The last part of my YT URL
In Title: The title of the 1st video in my list
In Type: Nothing can be filled in.
In SEF URL: Auto generated according to title.
Below (which says LAN_YOUTUBE_ADMIN_06): The full URL of my YT channel.

The result is that I get one title showing, and when I click on that, I get a preview of all my videos in my channel, and on the bottom it says "LAN_YOUTUBE_001, and underneath that there is a red exclamation mark with "Error" next to it.

I have a feeling a few things are not 100% correct, I also have an API key for the YT embed API and the YT v3 API, but nothing asks me to fill that in anywhere, and there is hardly any documentation, so if someone could enlighten me in as plain as possible English, I would be truly grateful.
My e107 blog can be found at https://www.vlaamseradio.tk/blog/ or https://www.vlaamseradio.uk/blog/

Thank you in advance.

**Comment by tgtje** [comment]
(( having LAN's displayed means language issues)
Well just an answer alongside the question. If you like a listing of available "movies as link" that can be easily achieved.
I have also done some tinkering with the YT plugin, helas no success (maybe i do miss something??).
But i am using the News Feeds (rss) function.
Now news feeds puts a link in the nav bar (container for feeds), (rename possible) , just add the YT rss feed from your channel in it : like
`www.youtube.com/feeds/videos.xml?channel_id=yourchannelid    < numb+chars`  done . a nice list of urls will be displayed (GDPR?  = 3rd party cookie!). But wait : it is still a list form in a construction, and maybe you do not like that....
Solution > create the newsfeed ..make it so.. Next step : your yt feed is a 'sub' inside the feeds section.
Now copy the url from that feed (hover and click copy url) and create a new link inside navigation (your name choice....).
This will result in a kind of 'standalone page.
Will this be the only feed on site?; hide the navbar News Feeds (eg. class= nobody).  Only yours should have a  link now.. (ala like a page).
Done

**Comment by Jimmi08** [comment]
first question -  did you install it from e107.org (or admin manager) or from GitHub repo?  
[https://github.com/e107inc/youtube](https://github.com/e107inc/youtube)

**Reply by Jimmi08** [reply]
Github version works without issues, tested only user channel without YTB API Key.
